spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -141,5 +177,83 @@ module.exports = function (provider, setup) {
     });
   });
 
-  // TODO: add tests for tcpsocket.secure, accepting multiple.
+  it("Secures a socket", function (done) {
+    socket.connect('www.google.com', 80, function () {
+      var prepared = false;
+      dispatch.gotMessageAsync('onData', [], function (msg) {
+        expect(prepared).toEqual(true);
+        socket.secure(done);
+      });
+      // prepareSecure is Chrome specific, see freedom-for-chrome
+      socket.prepareSecure(function () {
+        prepared = true;
+        socket.write(rawStringToBuffer('GET / HTTP/1.0\n\n'),
```

Comment:
Good call, I'll switch to port 443 and see if I can get more substantive `expect` statements.